database/delete.py

```diff
@@ -0,0 +1,23 @@
+import pymysql
+from database.connection import db_connect
+
+def db_delete(db):
+    cursor=db.cursor()
+    delete_tbl="delete from tbl_employee1 where Status= 'Inactive';"
+    delete_val= cursor.execute(delete_tbl)
+    print(delete_val)
+    if delete_val >0:
```

Comment:
Same as previous function comment. Create separate function for validation.

Reply:
For validation ,every function has their different conditions to validate in it. 
lets discuss on this call

Reply:
I mean create different validation function for each action (create, insert, select, update, delete)